docs/resources/task.md

```diff
@@ -56,7 +56,7 @@ resource "iterative_task" "example" {
 
 - `region` - (Optional) [Cloud region/zone](#cloud-region) to run the task on.
 - `machine` - (Optional) See [Machine Types](#machine-type) below.
-- `disk_size` - (Optional) Size of the ephemeral machine storage in GB.
+- `disk_size` - (Optional) Size of the ephemeral machine storage in GB. `-1`: automatic based on `image`.
```

Comment:
automatic means what? Guessing image but not sure.

Reply:
Agree. More than one might pitfall in not having enough space for the project. 
How does TPI handle not having enough space when syncing data? 🤔

Reply:
Automatic `disk_size = image_size`

Reply:
It simplifies considerably the basic use cases, and users can always increase it. Is either that or putting `disk_size = 150` on all the examples.

Reply:
> It simplifies considerably the basic use cases, and users can always increase it. Is either that or putting disk_size = 150 on all the examples.

I find it better to set a min/default size due to what I say. Users do an example and suddenly it does not work due to sync data

Reply:
> _I find it better to set a min/default size_

Which would be 150 GB for cross–provider compatibility with the official machine images.

Reply:
Is this change indeed not breaking anything?

Reply:
Tests are green

Reply:
Speaking about examples / tutorials

Reply:
> _Suddenly it does not work due to sync data_

It's a risk, of course, but we can't account for the data needs of every user 🙃

Reply:
> _Speaking about examples / tutorials_

Nope. This change is fully backwards compatible for all the examples/tutorials I am aware of.

Reply:
In fact, this pull request is the only way for the Jupyter example to work on every cloud

Reply:
Resolved with https://github.com/iterative/terraform-provider-iterative/pull/543#issuecomment-1111031249; free space is not a concern at all.